modules/id5IdSystem.js

```diff
@@ -9,6 +9,42 @@ import * as utils from '../src/utils'
 import {ajax} from '../src/ajax';
 import {submodule} from '../src/hook';
 
+const ID5_ENDPOINT = 'https://id5-sync.com/g/v1';
+
+function buildUrl(config, consentData, storedUserId) {
```

Comment:
There is already a url.format to do url formatting in prebid. If you want to refactor this code, the third parameter of `ajax` is an object with query param. I think it would be better.

Reply:
hi @padurgeat I pushed the change for this one, let me know if it looks good. thanks.